tests/testthat.R

```diff
@@ -0,0 +1,12 @@
+# This file is part of the standard setup for testthat.
+# It is recommended that you do not modify it.
+#
+# Where should you do additional test configuration?
+# Learn more about the roles of various files in:
+# * https://r-pkgs.org/tests.html
+# * https://testthat.r-lib.org/reference/test_package.html#special-files
+
+library(testthat)
+library(tfpscanner)
+
+test_check("tfpscanner")
```

Comment:
Current tests are empty but I can see that they're added in the PR that comes after this one, so that's fine :)